HyPDF.text.rb

```diff
@@ -0,0 +1,7 @@
+File.open('public/down-the-rabbit-hole.txt', 'w:UTF-8') {|f| f.write(@hypdf = HyPDF.pdftotext(
```

Comment:
Line is too long. [94/80]<br>Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.